### ISSUE TITLE
Client fails silently if log directory does not exist

### ISSUE THREAD

**Issue by rdeago** [cloudradar-monitoring/rport#860]
**Describe the bug**

On a Raspberry Pi 4 with Debian 11, I have mounted `/var/log` on tmpfs to avoid writing to the Micro SD card, so obviously when `rport` starts there is no `/var/log/rport` directory.

Instead of creating the directory, as AFAIK practically all other programs do, RPort fails silently (`service rport status` reports `(code=exited, status=1/FAILURE)`).

If I create the `/var/log/rport` directory and start RPort again (`service rport start`), RPort runs without problems.

**Environment**

* Raspberry Pi 4
* Debian Linux 11 (Raspberry Pi OS Lite)
* No Docker, just the plain OS and a couple utility programs (Midnight Commander, Tilde editor)
* Both RPort versions 0.9.0 and 0.9.5 exhibit this behavior

**Log file**

On the client, _ça va sans dire_, no log file to speak about.

On the server, nothing relevant to the problem at hand.

**Screenshots**

**Additional context**

This is how I create the log directory:

```shell
sudo mkdir /var/log/rport
sudo chown rport:rport /var/log/rport
```

**Comment by thorstenkramm** [comment]
@rdeago Rport cannot create this directory because rport starts without root rights. Only root can create directories under `/var/log`. Other daemons start as root and give up root rights later. But rport is not using this approach.

You can solve this problem by instructing systemd to create the directory if missing before starting rport.
Insert a [`ExecStartPre`](https://www.freedesktop.org/software/systemd/man/systemd.service.html#ExecStartPre=) command to `/etc/systemd/system/rport` . Also insert `PermissionsStartOnly=true` so the pre start script is executed by root.
```
[Unit]
Description=Create reverse tunnels with ease.
ConditionFileIsExecutable=/usr/local/bin/rport

[Service]
StartLimitInterval=5
StartLimitBurst=10
ExecStart=/usr/local/bin/rport "-c" "/etc/rport/rport.conf"
ExecStartPre=/bin/sh -c "test -e /var/log/rport/||(mkdir /var/log/rport&&chown rport /var/log/rport)"
PermissionsStartOnly=true
User=rport
Restart=always
RestartSec=120
EnvironmentFile=-/etc/sysconfig/rport

[Install]
WantedBy=multi-user.target
```

**Comment by rdeago** [comment]
@thorstenkramm while I'm of course grateful for your suggestion, I can't help noticing that what you're basically saying is: "This simple modification to a file that is part of our software distribution would solve the problem, but we won't do it".

Would you at least accept a PR?
